src/cljs/harja/views/urakka/paallystyskohteet.cljs

```diff
@@ -199,20 +202,24 @@
           ur
           paallystys-tiedot/yhan-paallystyskohteet
           {:otsikko (if @tallennus-gif?
-                      [ajax-loader-pieni yha-kohteet-otsikko]
-                      yha-kohteet-otsikko)
+                      [ajax-loader-pieni (yha-kohteet-otsikko @urakka/valittu-urakan-vuosi)]
+                      (yha-kohteet-otsikko @urakka/valittu-urakan-vuosi))
            :kohdetyyppi :paallystys
            :yha-sidottu? true
+           :piilota-tallennus? (when (< @urakka/valittu-urakan-vuosi 2020) true) ;; 2020 kaistamuutosta edeltäviä kohteita ei saa enää muokata.
            :tallenna (when (oikeudet/voi-kirjoittaa? oikeudet/urakat-kohdeluettelo-paallystyskohteet (:id ur))
                        tallenna-kohde)
            :kun-onnistuu kohteen-tallennus-onnistui}]
 
          [yllapitokohteet-view/yllapitokohteet
           ur
           paallystys-tiedot/harjan-paikkauskohteet
-          {:otsikko      "Harjan paikkauskohteet ja muut kohteet"
+          {:otsikko      (if (< @urakka/valittu-urakan-vuosi 2020)
```

Comment:
Sama tarkistus tehdään kolme kertaa `(< @urakka/valittu-urakan-vuosi 2020)`. Voisi olla hyvä laittaa `let`iin tuo arvo ja käyttää sitä. Muutenkin tuollaisten predikaattien tulokset on ihan hyvä laittaa, jonkun muuttujan taakse ja antaa sille muuttujalle tarkoituksen mukainen nimi, niin muilla lukijoilla on helppo päästä kärryille, että miksi näin tehdään. Tässä nyt ei sinänsä ole niin paha, kun tuo predikaatti on yksinkertainen, alla olevasta stringistä selviää vähän ja rivillä 222 rivillä on kommentoitu asiaa (jay, kommentit on kivoja :))